projects/ngrx.io/content/navigation.json

```diff
@@ -455,16 +455,21 @@
         }
       ]
     },
+    {
+      "title": "ESLint Plugin",
```

Comment:
I was thinking of making this a direct menu item, is that OK?
It would have the following sub-pages:

- Overview - list of all rules and configurations
- Installation - more or less the current page with installation instructions

Reply:
I'd add "Developer Tooling" category in the menu at the root level, based on this categorization:

![ngrx-categories](https://user-images.githubusercontent.com/17877290/163804212-619a529e-195e-4cd9-93ca-16d86cc26deb.PNG)

We can then add `@ngrx/eslint-plugin` and move `@ngrx/store-devtools` and `@ngrx/schematics` from "State" to "Developer Tooling" menu:

- Developer Tooling
  - @ngrx/store-devtools
  - @ngrx/schematics
  - @ngrx/eslint-plugin

Thoughts? 👀

Reply:
Thanks a great idea, I like it!

Reply:
Agreed!

Reply:
The new preview uses the suggested navbar 🤩
![image](https://user-images.githubusercontent.com/28659384/164284338-f58757ae-5394-48d4-acbd-ad0dcf9ab476.png)

Reply:
I went for developer tools though, what's your preference?

Reply:
I got used to the name "developer tooling packages". 😅 Developer tools also sound good to me.